lib/redux/rootReducer.ts

```diff
@@ -1,12 +1,14 @@
 /* Instruments */
 import { combineReducers } from 'redux';
 import GPSReducer from '@/stores/GPS/GPSReducers';
+import GlobalPathReducer from '@/stores/GlobalPath/GlobalPathReducers';
 import AISShipsReducer from '@/stores/AISShips/AISShipsReducers';
 
 export default () => {
     const reducerMap = {
         gps: new GPSReducer().reducer,
-        aisships: new AISShipsReducer().reducer,
+        globalpath: new GlobalPathReducer().reducer,
```

Comment:
One more adjustment: Can you camelcase here too please `globalpath` -> `globalPath`? I noticed we have some inconsistencies with `aisships`, I'm going to address to Jordan later as well.

Thanks for your patience